create_venv.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+The script can be used to setup a virtual environment for running Firefox UI Tests.
+It will automatically install the firefox ui test package, all its dependencies, and
+auxiliary packages like mozdownload.
+"""
```

Comment:
I don't understand what dependencies are required and which are optional.  It looks like mozdownload is optional.